src/test/ui/did_you_mean/issue-39802-show-5-trait-impls.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+trait Foo<B> {
+    fn bar(&self){}
+}
+
+impl Foo<u8> for i8 {}
+impl Foo<u16> for i8 {}
+impl Foo<u32> for i8 {}
+impl Foo<u64> for i8 {}
+impl Foo<bool> for i8 {}
+
+fn main() {
+    Foo::<i32>::bar(&1i8);
+
+    //~^ error: the trait `Foo<i32>` is not implemented for `i8``
```

Comment:
You are right, I copied it from `src/test/ui/did_you_mean/issue-21659-show-relevant-trait-impls-1.rs` which has both but I did not noticed the stderr file.